docs/section9/Rule6-aa.md

```diff
@@ -0,0 +1,57 @@
+
+# Lighting - Rule 6-aa
+
+**Rule ID:** 6-aa  
+**Rule Description:** Proposed building is modeled with other programmable lighting controls through a 10% schedule reduction.  
+**Rule Assertion:** Proposed RMR = expected value  
+**Appendix G Section:** Lighting  
```

Comment:
Table G3.1 Part 6 Lighting under Proposed Building Performance paragraph (i)

Reply:
Updated in #39